plugin/internal/remotecall/types.go

```diff
@@ -0,0 +1,41 @@
+package remotecall
+
+import "github.com/golang-jwt/jwt"
+
+type userKeyAuthRequest struct {
+	AccountId    string       `json:"accountIdentifier"`
```

Comment:
we usually use snake casing, any reason to deviate here?

Reply:
its coming from harness core which does other way :(